torch/quantization/observer.py

```diff
@@ -841,8 +843,9 @@ def calculate_qparams(self):
 
     def _save_to_state_dict(self, destination, prefix, keep_vars):
         super(HistogramObserver, self)._save_to_state_dict(destination, prefix, keep_vars)
-        destination[prefix + 'min_val'] = self.min_val
-        destination[prefix + 'max_val'] = self.max_val
+        if self.min_val is not None and self.max_val is not None:
```

Comment:
Will this cause an issue with loading from state dict if strict=True? In that case, we will not have entries for min and max val which can trigger an error.

Reply:
I think we can make this key optional by excluding them from missing keys